commands/create_kubeconfig_test.go

```diff
@@ -5,11 +5,48 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
+	"os"
+	"path"
+	"strings"
 	"testing"
 
 	"github.com/giantswarm/gsctl/config"
 )
 
+// create a temporary directory
+func tempDir() string {
+	dir, _ := ioutil.TempDir("", config.ProgramName)
+	return dir
+}
+
+// create a temporary kubectl config file
+func tempKubeconfig() (string, error) {
+
+	// override standard paths for testing
+	dir := tempDir()
+	config.HomeDirPath = dir
+	config.DefaultConfigDirPath = path.Join(config.HomeDirPath, ".config", config.ProgramName)
+	config.LegacyConfigDirPath = path.Join(config.HomeDirPath, "."+config.ProgramName)
+
+	// add a test kubectl config file
+	kubeconfigpath := path.Join(dir, "tempkubeconfig")
```

Comment:
Nitpick but the casing of kubeconfigpath is confusing me. Is it so it doesn't conflict with kubeConfigPath?

Sorry I didn't get back to you on the previous PR. I also couldn't think of a better name! Naming is definitely hard and especially on Fridays!